### ISSUE TITLE
Allow the players zoom to be altered in the launcher

### ISSUE THREAD

**Issue by Cepheid-UK** [w3champions/launcher#88]
A few players have mentioned the auto-zoom reset, I think it might be nice if we could have the setting for your zoom be displayed on the launcher.

The launcher could read your text file and see what it's set to, display like:

Player Zoom - 1750
Observer Zoom - 2000

If we can detect when the player is in a game as an observer, we could have the launcher edit the file to change their zoom to whatever they have in the launcher, when the next game starts as a player, the text file could be re-written as their player zoom value.

This would be a workaround to allow observers to change their zoom level by alt-tabbing to the launcher and changing the value, which would update the next 15s tick.

**Comment by modmoto** [comment]
hmm, seems like a lot of stuff that has to be considered for a workaround. Isnt it better to do that in the maps somehow?

**Comment by Cepheid-UK** [comment]
The problem is the maps dont allow observers to interact with the map triggers. That's why a different solution is needed. There may be another way to do this though.

**Comment by Cepheid-UK** [comment]
At the least, would it be beneficial to allow users to edit their zoom on the launcher? In theory that's just editing the value of their text file, right?

**Comment by modmoto** [comment]
Yes that is simple, but I wanted to move those kind of options to the ingame client. I think hotkeys are fine in the launcher, but other options should go ingame, so we have only one place. For example, I wanted to add the skins ingame aswell. We can foward messages to the launcher to make file read/write operations.

**Comment by modmoto** [comment]
I think the idea is good, but I dont know if this is the best way to do it, imo.
